### PR TITLE
Aws ec2 commands

### PR DESCRIPTION
**Pull request by Shellyber**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://github.com/demisto/etc/issues/35969

## Description
Added 2 new commands: 
- aws-ec2-allocate-hosts
- aws-ec2-release-hosts

## Minimum version of Demisto
- [x] 5.0.0

## Does it break backward compatibility?
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by Shellyber** [comment]
@roysagi Please review the rn

File: Packs/AWS-EC2/Integrations/AWS-EC2/AWS-EC2.yml [hunk 1]
```diff
@@ -6475,85 +6475,285 @@ script:
...
+      required: false
+      secret: false
+    deprecated: false
     description: Removes egress rule from a security group. To remove a rule, the
       values that you specify (for example, ports) must match the existing rule's
       values exactly.
     execution: true
     name: aws-ec2-revoke-security-group-egress-rule
-  dockerimage: demisto/boto3:2.0.0.18824
+  - arguments:
```
Comment (Itay4) [inline Packs/AWS-EC2/Integrations/AWS-EC2/AWS-EC2.yml:6636]: arent you missing here the client args (region, roleArn, etc..)?

File: Packs/AWS-EC2/Integrations/AWS-EC2/AWS-EC2.yml [hunk 2]
```diff
@@ -6475,85 +6475,285 @@ script:
...
+    execution: false
+    name: aws-ec2-allocate-hosts
+    outputs:
+    - contextPath: AWS.EC2.Host.HostId
+      description: The ID of the allocated Dedicated Host. This is used to launch
+        an instance onto a specific host.
+      type: String
+  - arguments:
+    - default: false
+      description: The IDs of the Dedicated Hosts to release.
```
Comment (Itay4) [inline Packs/AWS-EC2/Integrations/AWS-EC2/AWS-EC2.yml:6713]:
```suggestion
      description: Comma-seperated of IDs of the Dedicated Hosts to release.
```

File: Packs/AWS-EC2/Integrations/AWS-EC2/AWS-EC2.py [hunk 3]
```diff
@@ -2944,6 +2944,50 @@ def create_traffic_mirror_session_command(args):
...
+    if args.get('instance_type'):
+        kwargs.update({'InstanceType': args.get('instance_type')})
+    if args.get('instance_family'):
+        kwargs.update({'InstanceFamily': args.get('instance_family')})
+    if args.get('host_recovery'):
+        kwargs.update({'HostRecovery': args.get('host_recovery')})
+
+    response = client.allocate_hosts(AvailabilityZone=availability_zone, Quantity=quantity, **kwargs)
+    data = ({
+        'HostId': response['HostIds']
```
Comment (Itay4) [inline Packs/AWS-EC2/Integrations/AWS-EC2/AWS-EC2.py:2971]:
```suggestion
        'HostId': response.get('HostIds')
```

File: Packs/AWS-EC2/Integrations/AWS-EC2/AWS-EC2.py [hunk 4]
```diff
@@ -2944,6 +2944,50 @@ def create_traffic_mirror_session_command(args):
...
+    client = aws_session(
+        region=args.get('region'),
+        roleArn=args.get('roleArn'),
+        roleSessionName=args.get('roleSessionName'),
+        roleSessionDuration=args.get('roleSessionDuration'),
+    )
+    host_id = argToList(args.get('host_id'))
+    response = client.release_hosts(HostIds=host_id)
+    if response['ResponseMetadata']['HTTPStatusCode'] == 200:
+        demisto.results("The ID of the Dedicated Hosts was successfully released.")
```
Comment (Itay4) [inline Packs/AWS-EC2/Integrations/AWS-EC2/AWS-EC2.py:2988]: shouldnt be something like `The host was successfully released` ?

**Review by Itay4 — COMMENTED** [review]
looks good
please add the new commands to the README and the command_examples file

File: Packs/AWS-EC2/Integrations/AWS-EC2/AWS-EC2.yml (hunk 1 above)
Comment (Shellyber) [reply Packs/AWS-EC2/Integrations/AWS-EC2/AWS-EC2.yml:6636]: I'll add them, but I didn't add this command

Comment (Itay4) [reply Packs/AWS-EC2/Integrations/AWS-EC2/AWS-EC2.yml:6636]: arent those the arguments of aws-ec2-allocate-hosts ?

Comment (Shellyber) [reply Packs/AWS-EC2/Integrations/AWS-EC2/AWS-EC2.yml:6636]: yes sorry, got confused with the command above. 
Added them

**Review by Itay4 — APPROVED** [review]

File: Packs/AWS-EC2/Integrations/AWS-EC2/AWS-EC2.yml [hunk 5]
```diff
@@ -6475,85 +6475,311 @@ script:
...
+    execution: false
+    name: aws-ec2-allocate-hosts
+    outputs:
+    - contextPath: AWS.EC2.Host.HostId
+      description: The ID of the allocated Dedicated Host. This is used to launch
+        an instance onto a specific host.
+      type: String
+  - arguments:
+    - default: false
+      description: Comma-seperated of IDs of the Dedicated Hosts to release.
```
Comment (roysagi) [inline Packs/AWS-EC2/Integrations/AWS-EC2/AWS-EC2.yml:6739]:
```suggestion
      description: A comma-separated list of IDs of the Dedicated Hosts to release.
```

File: Packs/AWS-EC2/Integrations/AWS-EC2/README.md [hunk 6]
```diff
@@ -3059,3 +3059,75 @@ Creates a Traffic Mirror session.
...
+
+
+#### Base Command
+
+`aws-ec2-release-hosts`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| host_id | Comma-seperated of IDs of the Dedicated Hosts to release. | Required |
```
Comment (roysagi) [inline Packs/AWS-EC2/Integrations/AWS-EC2/README.md:3118]:
```suggestion
| host_id | A comma-separated list of IDs of the Dedicated Hosts to release. | Required | 
```

**Pull request merged**